src/rgw/rgw_rest_config.h

```diff
@@ -1,88 +0,0 @@
-// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*- 
-// vim: ts=8 sw=2 smarttab ft=cpp
-
-/*
- * Ceph - scalable distributed file system
- *
- * Copyright (C) 2013 eNovance SAS <licensing@enovance.com>
- *
- * This is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License version 2.1, as published by the Free Software
- * Foundation. See file COPYING.
- *
- */
-
-#pragma once
-
-#include "rgw_auth_s3.h"
-#include "rgw_rest.h"
-#include "rgw_zone.h"
-
-class RGWOp_ZoneGroupMap_Get : public RGWRESTOp {
-  RGWZoneGroupMap zonegroup_map;
-  bool old_format;
-public:
-  explicit RGWOp_ZoneGroupMap_Get(bool _old_format):old_format(_old_format) {}
-  ~RGWOp_ZoneGroupMap_Get() override {}
-
-  int check_caps(const RGWUserCaps& caps) override {
-    return caps.check_cap("zone", RGW_CAP_READ);
-  }
-  int verify_permission(optional_yield) override {
-    return check_caps(s->user->get_caps());
-  }
-  void execute(optional_yield y) override;
-  void send_response() override;
-  const char* name() const override {
-    if (old_format) {
-      return "get_region_map";
-    } else {
-      return "get_zonegroup_map";
-    }
-  }
-};
-
-class RGWOp_ZoneConfig_Get : public RGWRESTOp {
-  RGWZoneParams zone_params;
```

Comment:
oops, the ragweed tests are still using this `/admin/config` interface to read the local zone params